src/inquirer/render/console/_checkbox.py

```diff
@@ -10,9 +10,14 @@
 class Checkbox(BaseConsoleRender):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.selection = [k for (k, v) in enumerate(self.question.choices) if v in (self.question.default or [])]
+        self.locked = self.question.locked or []
+        self.selection = [k for (k, v) in enumerate(self.question.choices) if v in self.set_default_choices()]
         self.current = 0
 
+    def set_default_choices(self):
```

Comment:
this function is named strangely. I assume you wanted two make its return type clear?

However like this its named as a _Setter_-Methode, which it is clearly not. 
```suggestion
    def default_choices(self):
```

Reply:
Renamed